blockscout-ens/bens-logic/src/protocols/protocoler.rs

```diff
@@ -288,17 +290,114 @@ impl Protocoler {
         name: &str,
         network_id: i64,
         maybe_filter: Option<NonEmpty<String>>,
+    ) -> Result<Vec<DomainNameOnProtocol>, ProtocolError> {
+        let tlds = self
```

Comment:
Does it make sense to move it into else branch as it is not needed inside `if name.contains('.')` true condition?